spec/controllers/spree/admin/tax_rates_controller_spec.rb

```diff
@@ -29,7 +34,7 @@ module Admin
             end
           end
 
-          context "when the amount is changed" do
+          context "when the amount or included flag are changed" do
```

Comment:
I'm missing a spec that checks that `included_in_price` is indeed updated.

Reply:
Updated :ok_hand: